Source/Core/Common/ChunkFile.h

```diff
@@ -35,20 +35,8 @@
 
 // ewww
 
-#ifndef __has_feature
-#define __has_feature(x) (0)
-#endif
-
-#if (__has_feature(is_trivially_copyable) &&                                                       \
-     (defined(_LIBCPP_VERSION) || defined(__GLIBCXX__))) ||                                        \
-    (defined(__GNUC__) && __GNUC__ >= 5) || defined(_MSC_VER)
 #define IsTriviallyCopyable(T)                                                                     \
```

Comment:
In the spirit of cleanups, this should probably be a templated variable.

Reply:
Done.